ova/generate_ova.sh

```diff
@@ -155,6 +161,10 @@ function main() {
       clean
       exit 0
       ;;
+    "-k" | "--checksum")
+       cheksum="yes"
```

Comment:
Typo => `cheksum`.

Reply:
@DFolchA

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/5a72389d9ddbedad13d02074112ad463a9d9e133